jerry-core/ecma/base/ecma-globals.h

```diff
@@ -2148,6 +2148,41 @@ typedef struct
   uint32_t lazy_string_named_props; /**< number of lazy instantiated properties */
 } ecma_property_counter_t;
 
+/**
+ * Arguments object related status flags.
+ */
+typedef enum
+{
+  ECMA_ARGUMENTS_OBJECT_NO_FLAGS = 0,                    /* unmapped arguments object */
+  ECMA_ARGUMENTS_OBJECT_MAPPED = (1 << 0),               /* mapped arguments object */
+  ECMA_ARGUMENTS_OBJECT_CALLEE_INITIALIZED = (1 << 1),   /* 'callee' property has been lazy initialized */
+  ECMA_ARGUMENTS_OBJECT_CALLER_INITIALIZED = (1 << 2),   /* 'caller' property has been lazy initialized */
+  ECMA_ARGUMENTS_OBJECT_LENGTH_INITIALIZED = (1 << 3),   /* 'length' property has been lazy initialized */
+  ECMA_ARGUMENTS_OBJECT_ITERATOR_INITIALIZED = (1 << 4), /* 'Symbol.iterator' property has been lazy initialized */
+} ecma_arguments_object_flags_t;
+
+/**
+ * Definition of unmapped arguments object
```

Comment:
Sometimes there is a dot after sentences, sometimes not. Please make it consistent.